lib/git.zsh

```diff
@@ -12,10 +12,27 @@ function __git_prompt_git() {
 # Outputs current branch info in prompt format
 function git_prompt_info() {
   local ref
-  if [[ "$(__git_prompt_git config --get oh-my-zsh.hide-status 2>/dev/null)" != "1" ]]; then
-    ref=$(__git_prompt_git symbolic-ref HEAD 2> /dev/null) || \
-    ref=$(__git_prompt_git rev-parse --short HEAD 2> /dev/null) || return 0
-    echo "$ZSH_THEME_GIT_PROMPT_PREFIX${ref#refs/heads/}$(parse_git_dirty)$ZSH_THEME_GIT_PROMPT_SUFFIX"
+  if [[ "$(command git config --get oh-my-zsh.hide-status 2>/dev/null)" != "1" ]]; then
+    branches=$(command git branch --list -a 2> /dev/null)
+    if [[ "$branches" == "" && "$?" == "0" ]]; then
+        # Look for initialized folders
+        info="<initialized>"
```

Comment:
Does this work the same way as before? Now it'll just say `<initialized>`, before it says whatever the `git symbolic-ref HEAD` command returns, which is normally `master`.

Reply:
I added back the branch name to be displayed even on `<initialized>` state.